### PR TITLE
add webpage for dashboard

### PR DESCRIPTION
**Pull request by loin3**
스크린샷 2023-06-27 오후 3 26 46

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
How do I test the web page? 
Is it need a special node version?
I can't test this web.

File: dashboard/src/logo.svg
Comment (zemyblue) [inline dashboard/src/logo.svg:1]: How about using finschia logo image?

File: dashboard/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (zemyblue) [inline dashboard/public/manifest.json]: `Finschia HackWasm 2023`

File: dashboard/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (zemyblue) [inline dashboard/public/index.html]: `Finschia HackWasm2023`

File: dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+## Start IBC chains before run dashboard
+
+1. Clone [ibc relayer](https://github.com/loin3/relayer/tree/enable-rest)
+2. Follow
+   [demo](https://github.com/loin3/relayer/blob/enable-rest/docs/rest-open-lbmsim-lbmsim-demo.md).
```
Comment (zemyblue) [inline dashboard/README.md:5]: Please check this url.
I think it's good to use official url.

File: dashboard/src/logo.svg
Comment (loin3) [reply dashboard/src/logo.svg:1]: This image is not used currently.
I will delete this image.

**Review by zemyblue — DISMISSED** [review]

**Review by tkxkd0159 — COMMENTED** [review]
Do we need to have react logo? (logo192, logo512)

**Comment by tkxkd0159** [comment]
Please check package vulnerabilities & favicon. I think you can get the favicon from the marketing team that we're already using at Finschia Docs.

**Comment by tkxkd0159** [comment]
both chains can't be registered on DOSI Vault simultaneously. Please check it.

**Comment by loin3** [comment]
When receiving a suggest chain from dosi vault, it is determined whether the same chain id exists. If there is, the suggest chain doesn't work. 
However, the criterion for determining whether the chain id is the same is slightly different. For example, if there is a chain ID such as `finschia-0`, the ID to determine if there is an identical chain is `finschia`. So, if you try to suggest `finschia-1`, it fails because the chain ID is the same as `fisnchia`.

**Review by tkxkd0159 — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**